build/config/sysroot.gni

```diff
@@ -27,8 +27,12 @@ if (current_toolchain == default_toolchain && target_sysroot != "") {
 } else if (is_linux && !is_chromeos) {
   if (current_cpu == "mipsel") {
     sysroot = rebase_path("//mipsel-sysroot/sysroot")
-  } else if (use_default_linux_sysroot && !is_fuchsia && current_cpu == "x64") {
-    sysroot = rebase_path("//build/linux/debian_sid_amd64-sysroot")
+  } else if (use_default_linux_sysroot && !is_fuchsia) {
+    if (current_cpu == "x64") {
+      sysroot = rebase_path("//build/linux/debian_sid_amd64-sysroot")
+    } else {
+      sysroot = rebase_path("//build/linux/debian_sid_arm64-sysroot")
```

Comment:
I'm a little confused by the scope of this PR; the title says "Host", but this change appears to be specific to building the target artifacts. And the "Tests" section (which isn't actually discussing tests at all) appears to be showing a manual end-to-end process of some kind but I'm not clear on whether it's host, target, or both.

Please update the PR description to clearly describe what exactly this PR is for. It appears to me from the changes that the answer is that it's for cross-compiling the engine on Linux.

Reply:
Thank you.
It's my mistake. target_cpu is correct.. Please wait for a while.

Reply:
`current_cpu` looks correct to me in this usage; my confusion was around the description of the PR vs what it seems to be doing.

Reply:
I updated the description. Could you check that?